packages/components/src/animate/index.js

```diff
@@ -23,13 +53,29 @@ export function useAnimate( { type, origin = getDefaultOrigin( type ) } ) {
 		);
 	}
 
-	if ( type === 'loading' ) {
-		return classnames( 'components-animate__loading' );
-	}
+	return undefined;
 }
 
-export default function Animate( { type, options = {}, children } ) {
+/**
+ *
+ * @todo Remove options when deprecated handling has been removed.
+ *
+ * @param {{options?: any; children: (props: {className?: string})=> JSX.Element;} & UseAnimateOptions} props
+ * @return {JSX.Element} Element
+ */
+export default function Animate( {
+	type,
+	options: _deprecatedOptions,
+	children,
+	...options
+} ) {
+	if ( Boolean( _deprecatedOptions ) ) {
+		deprecated( '<Animate> options prop', {
+			version: '9.3',
+			hint: 'Pass options directly as props instead.',
+		} );
+	}
```

Comment:
> It's something that needs more updates, documentation need to be aligned:
> https://github.com/WordPress/gutenberg/tree/master/packages/components/src/animate
> 
> In general, I agree with the sentiment but in practice it might start printing those depreciations in the JS console if its usage isn't updated in Gutenberg. So I would be very careful changing the public API. It also needs to be announced as Dev Note when a new version of WordPress is released so plugins could update.

_-@gziolo in https://github.com/WordPress/gutenberg/pull/26176#discussion_r519275097_

Reply:
Thanks, @gziolo. What's required for a dev note?